base/applications/regedit/framewnd.c

```diff
@@ -1119,6 +1218,8 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
     case ID_VIEW_STATUSBAR:
         toggle_child(hWnd, LOWORD(wParam), hStatusBar);
         return TRUE;
+    case ID_FAVOURITES_ADDTOFAVOURITES:
+        return DialogBoxW(hInst, MAKEINTRESOURCEW(IDD_ADDFAVORITES), hWnd, AddToFavoritesDlgProc);
```

Comment:
The `_CmdWndProc` function returns a BOOL (and is really used as such, see [old line 1399](https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/regedit/framewnd.c;hb=48a0d8e012f77b5d7fb11d58141e1fdd5de27254#l1398)), but here you return other things (the last error code).
It is not useful to return the error code of the favourite dialog since

Reply:
Yes, my bad, I just assumed it was LRESULT.